src/bindings/c/tests/ov_auto_plugin_test.cpp

```diff
@@ -0,0 +1,62 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "ov_test.hpp"
+
+class ov_auto_plugin_test : public ov_capi_test_base {
+public:
+    void SetUp() override {
+        ov_capi_test_base::SetUp();
+    }
+
+    void TearDown() override {
+        ov_capi_test_base::TearDown();
+    }
+};
+
+TEST_P(ov_auto_plugin_test, ov_core_auto_set_and_get_property_bool) {
+    auto device_name = GetParam();
+    ov_core_t* core = nullptr;
+    OV_EXPECT_OK(ov_core_create(&core));
+    EXPECT_NE(nullptr, core);
+
+    const char* key_1 = ov_property_key_intel_auto_device_bind_buffer;
+    const char* key_2 = ov_property_key_intel_auto_enable_startup_fallback;
+    const char* key_3 = ov_property_key_intel_auto_enable_runtime_fallback;
+    const char* enable = "YES";
+    const char* disable = "NO";
+    OV_EXPECT_OK(ov_core_set_property(core, device_name.c_str(), key_1, enable));
+    char* ret = nullptr;
+    OV_EXPECT_OK(ov_core_get_property(core, device_name.c_str(), key_1, &ret));
```

Comment:
If we go through all (key, value) pairs, why not use unified INSTANTIATE_TEST_SUITE_P to do it?

Reply:
Yes, updated.